examples/sap_inventory_update/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ballerina generates this directory during the compilation of a package. 
```

Comment:
Same since we have gitignore files in parent directory

Reply:
Removed with Changed with [1587866](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/2/commits/1587866408961566b051f449be7fd7303ce85a9f).